dashboard/app/models/school_stats_by_year.rb

```diff
@@ -46,21 +46,31 @@ class SchoolStatsByYear < ActiveRecord::Base
 
   belongs_to :school
 
+  # Mapping of urban-centric locale code to community type.
+  COMMUNITY_TYPE_MAP = {
+    '11' => 'city_large',
+    '12' => 'city_midsize',
+    '13' => 'city_small',
+    '21' => 'suburban_large',
+    '22' => 'suburban_midsize',
+    '23' => 'suburban_small',
+    '31' => 'town_fringe',
+    '32' => 'town_distant',
+    '33' => 'town_remote',
+    '41' => 'rural_fringe',
+    '42' => 'rural_distant',
+    '43' => 'rural_remote'
+  }.freeze
```

Comment:
Are these values from NCES or some other external source? How about adding a comment pointing to the source?

Reply:
From NCES, added comment.